app/crm/worker_main.py

```
@@ -28,7 +29,10 @@
         claim_sends,
         dispatch_send,
         interval=CRM_WORKER_INTERVAL,
-        batch=CRM_WORKER_BATCH,
+        # Its own dial, not CRM_WORKER_BATCH: a claimed batch must finish
+        # inside the claim lease (batch × send timeout < lease), or another
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)